amaznot/app/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Search;
+use Illuminate\Database\QueryException;
+
+class SearchController extends Controller
+{
+    public function search(Request $request, $search = "")
+    {
+        try
+        {
+            if($request->wantsJson())
```

Comment:
When is this if condition true? Also, if the condition is removed, the json you return throws an exception.